docs/tutorials/getting-started-with-flask.rst

```diff
@@ -0,0 +1,454 @@
+Build a rock for a Flask application
+------------------------------------
+
+In this tutorial you will create a simple Flask application and learn how to
+containerise it in a rock, using Rockcraft's ``flask-framework``
+:ref:`extension <flask-framework-reference>`.
+
+Setup
+=====
+
+We recommend starting from a clean Ubuntu 22.04 installation. If you don't have
+one available, you can create one using Multipass_:
+
+.. collapse:: How to create an Ubuntu 22.04 VM with Multipass
+
+    Is Multipass_ already installed and active? Start by running
+
+    .. code-block:: bash
+
+        sudo snap services multipass
+
+    If you see the ``multipass`` service but it isn't "active", then you'll
+    need to run ``sudo snap start multipass``. On the other hand, if you get
+    an error saying ``snap "multipass" not found``, then you must Install
+    Multipass_:
+
+    .. code-block:: bash
+
+        sudo snap install multipass
+
+    Then you can create the VM with the following command:
+
+    .. code-block:: text
+
+        multipass launch --disk 10G --name flask-hello-world 22.04
+
+    Finally, once the VM is up, open a shell into it:
+
+    .. code-block:: bash
+
+        multipass shell flask-hello-world
+
+----
+
+LXD will be required for creating the OCI image. Make sure it is installed an
+initialised:
+
+.. code-block:: bash
+
+   sudo snap install lxd
```

Comment:
At least in the multipass VM, it is there by default.
```
ubuntu@flask-hello-world:~$ sudo snap install lxd
snap "lxd" is already installed, see 'snap help refresh'
```

Reply:
that's alright. It's an info message. harmless. I think that's less error prone than adding text saying "if you're on VM .... then it might already be installed, but you still need to `init` it with ...etc."